### PR TITLE
feat(Bookings): Display bookings in Calendar

### PR DESCRIPTION
**Pull request by paradoux**
- Fetch bookings for the visible month only
- Display each booking in the Calendar

### CODE REVIEW

File: packages/server/src/controllers/bookings.ts [hunk 1]
```diff
@@ -49,20 +51,48 @@ export const createBooking = async (ctx: Context) => {
...
+    console.log(error);
+    console.log(`Error while retrieving bookings by interval`);
+    ctx.status = 500;
+  }
+};
+
 export const getBookings = async (ctx: Context) => {
   try {
-    const bookings = await getAllBookings();
+    const bookings = await retrieveAllBookings();
```
Comment (AlexGeb) [inline packages/server/src/controllers/bookings.ts:91]: Why this renaming? What value does it bring ?

File: packages/server/src/controllers/bookings.ts [hunk 2]
```diff
@@ -49,20 +51,48 @@ export const createBooking = async (ctx: Context) => {
...
-    const booking = await getBookingById(bookingId);
+    const booking = await retrieveBookingById(bookingId);
     ctx.body = booking;
   } catch (error) {
     console.log(`Error while retrieving a booking: ${bookingId}`);
     ctx.status = 500;
   }
 };
 
+export const getBookingsByInterval = async (ctx: Context) => {
```
Comment (AlexGeb) [inline packages/server/src/controllers/bookings.ts:62]: I don't understand the `get..` and `retrieve..` namings, shouldn't we stick to a single naming convention?
I vote for `getEntit(y,ies)ByField` for the methods that could throw if the entity is not found and `findEntit(y,ies)ByField` for methods which returns `Entity | null`

File: packages/server/src/db/accounts.ts [hunk 3]
```diff
@@ -9,8 +9,8 @@ export const createOneAccount = async (
   return await knex("accounts")
     .returning(["account_id", "first_name", "last_name"])
     .insert({
-      firstName,
-      lastName,
+      first_name: firstName,
+      last_name: lastName,
```
Comment (AlexGeb) [inline packages/server/src/db/accounts.ts:13]: 🐛 sorry for this

File: packages/server/src/controllers/bookings.ts [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
   updateBookingById,
   BookingStatus,
+  retrieveBookingsByInterval,
 } from "../db/bookings";
 import {
   validateNewBookingData,
   createNewBooking,
   haveBookingDatesChanged,
 } from "../utils/booking";
+import { format, toDate } from "date-fns";
```
Comment (AlexGeb) [inline packages/server/src/controllers/bookings.ts:14]: To date is unused
```suggestion
import { format } from "date-fns";
```

Comment (AlexGeb) [reply packages/server/src/controllers/bookings.ts:14]: run yarn install on your project root to instal eslint.
You should also install the eslint extension for vscode to get errors directly in your editor

File: packages/server/src/controllers/bookings.ts (hunk 1 above)
Comment (paradoux) [reply packages/server/src/controllers/bookings.ts:91]: So it's in order to differentiate handler functions from database functions :)) 
For example for the route returning all the bookings inside an interval, the controller function is named getBookingsByInterval, this function performing some logic and calling another function retrieveBookingsByInterval which will fetch the bookings in the DB. I can't really name both functions with the same name so I use get* for controller functions and retrieve* for DB functions :))

File: packages/server/src/db/accounts.ts (hunk 3 above)
Comment (paradoux) [reply packages/server/src/db/accounts.ts:13]: No worries :))

File: packages/server/src/controllers/bookings.ts (hunk 4 above)
Comment (paradoux) [reply packages/server/src/controllers/bookings.ts:14]: Thanks!

File: packages/server/src/controllers/bookings.ts (hunk 2 above)
Comment (paradoux) [reply packages/server/src/controllers/bookings.ts:62]: See above :))

Comment (AlexGeb) [reply packages/server/src/controllers/bookings.ts:62]: we could suffix all request handlers by `handler`. Like: `getAllBookingsHandler` or `signupHandler`, WDYT ?

Comment (paradoux) [reply packages/server/src/controllers/bookings.ts:62]: Yes could do, but it's increasing the length of each name. 
I check to see if there is some conventions on this matter :))

**Pull request merged**